webpack.config.js

```diff
@@ -54,6 +54,10 @@ module.exports = (env, options) => {
     },
     module: {
       rules: [
+        {
+          test: /\.css$/,
+          use: ['style-loader', 'css-loader'],
```

Comment:
do we really need _both_? I don't really know what the difference is

Reply:
`css-loader` loads the CSS from its file as a string, whereas `style-loader` then injects it at runtime.

However, while investigating this I found that the injection gave us ~3KiB uncompressed bundle size in overhead, which is a lot for.. not much. Instead, I've switched to using `css-minimizer-webpack-plugin` for minimizing the created CSS, combined with `mini-css-extract-plugin` to inject it as a `<link>` tag in the `<head>` rather than using JavaScript.